Readme.md

```diff
@@ -3,22 +3,39 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E499)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How to show and hide a popup window via server side code
+# Popup Control for ASP.NET Web Forms - How to show and hide a popup window at the server side
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e499/)**
 <!-- run online end -->
 
+This example demonstrates how to change the popup control's visibility at the server side. Use this approach when you need to process data on the server before users open and/or after they close a popup window. Otherwise, we recommend to open and hide the popup window at the client to avoid a turn around to the server.
```

Comment:
```suggestion
This example demonstrates how to change the pop-up control's visibility on the server. Use this approach to process data on the server before users open and/or after they close a pop-up window. Otherwise, we recommend that you open and hide the pop-up window on the client to avoid a turn around to the server.
```

`a turn around` Can you explain?

Reply:
```suggestion
This example demonstrates how to change the Popup Control's visibility on the server. Use this approach when you need to process data on the server before users open and/or after they close a pop-up window. Otherwise, we recommend that you open and hide the pop-up window on the client to reduce the number of server round trips.
```

Reply:
...to reduce server round trips.

Reply:
```suggestion
This example demonstrates how to change the Popup Control's visibility on the server. Use this approach to process data on the server before users open and/or after they close a pop-up window. Otherwise, we recommend that you open and hide the pop-up window on the client to reduce server round trips.
```